src/Microsoft.Health.Fhir.Core/Features/Context/FhirRequestContextAccessor.cs

```diff
@@ -4,14 +4,15 @@
 // -------------------------------------------------------------------------------------------------
 
 using System.Threading;
+using Microsoft.Health.Core.Features.Context;
 
 namespace Microsoft.Health.Fhir.Core.Features.Context
 {
-    public class FhirRequestContextAccessor : IFhirRequestContextAccessor
+    public class FhirRequestContextAccessor : RequestContextAccessor<IFhirRequestContext>, IFhirRequestContextAccessor
```

Comment:
>IFhirRequestContextAccessor [](http://example.com/codeflow?start=91&length=27)

Can we kill it?
Generic question, doesn't have to be part of this PR, but can we not use it and then just remove the file?

It looks like pretty much same thing as another thing-y.

Reply:
Yea, we can probably stop using the `IFhirRequestContextAccessor`. I'll create a followup item to go revisit this here and in DICOM and see if it makes more sense to change the underlying type to an interface instead of an abstract class.